doc-content/enterprise-only/openshift/hacep-client-proc.adoc

```diff
@@ -5,7 +5,7 @@ You must adapt your CEP client code to communicate with the HA CEP server image.
 
 .Procedure
 
-.  Download the `{PRODUCT_FILE}-reference-implementation.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
+.  Download the `kogito-1.5.0-decision-services-quickstarts.zip` product deliverable file from the {PRODUCT_DOWNLOAD_LINK}[Software Downloads] page of the Red Hat Customer Portal.
```

Comment:
Are you sure? In CR1, rhdm-7.11.0-openshift-drools-hacep-distribution.zip is in rhdm-7.11.0-reference-implementation.zip.